sklearn/cluster/_spectral.py

```diff
@@ -21,6 +21,8 @@
 from ..neighbors import kneighbors_graph, NearestNeighbors
 from ..manifold import spectral_embedding
 from ._kmeans import k_means
+from ..utils._param_validation import Hidden, Interval, StrOptions, validate_params
```

Comment:
This line should be removed and you should only import `validate_params` in l.18 because this is the only function that has not been imported yet.